package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "leaflet": "^1.5.1",
```

Comment:
Nice! I'd think also about `https://www.npmjs.com/package/react-leaflet`, which might be more helpful later 😉